src/components/constants/display-sections.js

```diff
@@ -0,0 +1,2 @@
+const screenConstants = [];
+screenConstants['CHALLENGES_SCREEN_NAME'] = 'Challenges';
```

Comment:
What are you exporting here? How is this working in the file you imported? Please recheck @sumitd94